### PR TITLE
fix: restoring cache for pyproject.toml

### PR DESCRIPTION
**Pull request by nichmor**

### CODE REVIEW

File: test/pyproject-manifest/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+name = "pyproject-manifest"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+
+[tool.pixi.pypi-dependencies]
+boltons = "*"
```
Comment (pavelzw) [inline test/pyproject-manifest/pyproject.toml:19]:
```suggestion
boltons = "*"

```

**Review by pavelzw — APPROVED** [review]
LGTM as a quick fix. In general, we should make sure, though, that setup-pixi works out of the box if only a pyproject.toml is present without specifying manifest-path.

@nichmor can you bump the patch version in package.json? Then the release pipeline will be triggered automatically once merged.

**Comment by pavelzw** [comment]
Let's wait for #94 with merging, then we can release two fixes at once.

**Comment by pavelzw** [comment]
Closes #93

**Comment by nichmor** [comment]
> LGTM as a quick fix. In general, we should make sure, though, that setup-pixi works out of the box if only a pyproject.toml is present without specifying manifest-path.
> 
> @nichmor can you bump the patch version in package.json? Then the release pipeline will be triggered automatically once merged.

Hey!
Yeah, totally. I just wanted to make the scope of this PR very small and later maybe work on automatic discovery of pyproject manifest. 
Let me bump the patch version

**Comment by nichmor** [comment]
bumped.
@pavelzw can I merge it?

**Comment by pavelzw** [comment]
let's wait for CI and then we can merge it 👍🏻

**Pull request merged**